vyked/decorators/tcp.py

```diff
@@ -190,9 +190,20 @@ def wrapper(*args, **kwargs):
         logd = {
             'endpoint': func.__name__,
             'time_taken': end_time - start_time,
-            'hostname': hostname, 'service_name': service_name
+            'hostname': hostname,
+            'service_name': service_name,
+            'endpoint': func.__name__,
+            'api_execution_threshold_exceed': False
         }
-        logging.getLogger('stats').debug(logd)
+
+        method_execution_time = (end_time - start_time)
+
+        if method_execution_time > (CONFIG.API_THRESHOLD_TIMEOUT * 1000):
```

Comment:
keep API_THRESHOLD_TIMEOUT unit as millisecond

Reply:
TCP and http decorator work differently, one makes calculations in milliseconds other in seconds, so both ways it will work, and our all timeout have sec convention, so uniformity kept this in seconds